src/webgpu/api/validation/attachment_compatibility.spec.ts

```diff
@@ -135,6 +137,21 @@ class F extends ValidationTest {
       sampleCount,
     });
   }
+
+  getTextureFormatsRequiredExtensions(
+    formats: (DepthStencilFormat | undefined)[]
+  ): GPUExtensionName[] {
+    const extensions: GPUExtensionName[] = [];
+    for (const format of new Set(formats)) {
```

Comment:
Oops, I realized this isn't actually going to prevent duplicate extension names. My bad. Fixed in push.